backend/src/main/webapp/src/pages/__tests__/ApplicationPage.test.tsx

```diff
@@ -0,0 +1,8 @@
+// import * as React from 'react'
+// import {render, screen} from '@testing-library/react'
+// import { ApplicationPage } from '../ApplicationPage'
+// import { Table, TableComposable } from '@patternfly/react-table'
+
+// describe ('src/pages/ApplicationPage', () => {
+
+// })
```

Comment:
Tests are in a pending state for the whole UI. Lets work on this in a separate PR.